frontend/src/util/shared/survey.ts

```diff
@@ -11,28 +11,48 @@ export function mapToList<T>(map: Record<string, T>): SurveyOption[] {
 // count: number of students who chose the option
 export type OptionCount = {
```

Comment:
maybe we should rename `OptionCount` to `SurveyOption` because it contains more than a count now